Managers/MovableMan.cpp

```diff
@@ -781,21 +781,21 @@ Actor * MovableMan::GetUnassignedBrain(int team) const
 //                  best way to add it is. E.g. if it's an Actor, it will be added as such.
 //                  Ownership IS transferred!
 
-bool MovableMan::AddMO(MovableObject *pMOToAdd)
-{
-    if (!pMOToAdd)
+bool MovableMan::AddMO(MovableObject *pMOToAdd) {
+    if (!pMOToAdd) {
         return false;
+    }
 
     pMOToAdd->SetAsAddedToMovableMan();
 
     // Find out what kind it is and apply accordingly
-    if (Actor *pActor = dynamic_cast<Actor *>(pMOToAdd))
-    {
+    if (Actor *pActor = dynamic_cast<Actor *>(pMOToAdd)) {
```

Comment:
Data's code, not mine. I just moved the bracket up a line, I ain't cleaning up MovableMan (at least not yet)

Reply:
Yeah it looks like garbage but besides being convenient it's also more efficient I guess because it potentially saves you constructing an object and casting.
Gainzz over beauty, very tough decision.

Reply:
Does it save you actually making the object? You still have a `Thing *thing = dynamic_cast<Thing *>(blah)` line in there so I'm surprised if it does, but I haven't looked into it at all.

Reply:
Well yeah if the object is an actor, it never evaluates for device and thus doesn't construct and cast.
But if it's not actor ur fukd.

Reply:
Ohhh yeah, wasn't thinking about the context of the elseif here. You're right for sure, it's really good in this case, even if it's ugly as sin.